sdk/keyvault/azsecrets/version.go

```diff
@@ -0,0 +1,12 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azsecrets
+
+const (
+	moduleName = "azsecrets"
+	version    = "v0.8.0"
```

Comment:
Will the CI scripts get confused if there are two locations for `version`?

Reply:
Let's find out 😉 

If they do, I propose aligning on this pattern.

Reply:
We could, but I worry this will get us into trouble at some point as we now have two sources of truth that either must be kept in sync, or we know which is the "real" one.

Since we don't create a pipeline for data-plane packages, perhaps we just omit these constants entirely and leave managing them to the SDK author?  They aren't used by the generated code.

Reply:
Sounds good to me: https://github.com/Azure/autorest.go/issues/825